airsonic-main/src/main/java/org/airsonic/player/security/JWTAuthenticationProvider.java

```diff
@@ -75,13 +73,24 @@ private static boolean roughlyEqual(String expectedRaw, String requestedPathRaw)
                 return false;
             }
 
-            MapDifference<String, List<String>> difference = Maps.difference(expected.getQueryParams(),
-                    requested.getQueryParams());
+            Map<String, List<String>> left = new HashMap<>(expected.getQueryParams());
+            Map<String, List<String>> right = new HashMap<>(requested.getQueryParams());
+
+            /*
+                Here, test if the request come from the way but include the parameters size of request,
```

Comment:
This comment isn't in proper English, and despite my good will, I can't understand it :/
Are you trying to say something like:

> Remove the size of the request if it's present as a parameter: since Sonos is changing it, they can't ever be equal.